### ISSUE TITLE
Q: Status

### ISSUE THREAD

**Issue by ib00** [Sarroxxie/vk_procedural_displacement#19]
This looks like a cool project! Is this the implementation of the TFDM paper?

What's the current status? What's working and what's missing from the paper?

**Comment by Sarroxxie** [comment]
In short it is a mix of TFDM and the "High-Performance By-Example Noise using a Histogram-Preserving Blending Operator" paper (which creates procedural textures from an example pattern).
As this was part of my bachelor thesis, I wanted to get some distance from the project (as it was a bit stressful) but might come back to it in the future. 
Great parts of the project are only vaguely commented and I don't think it is intuitively to use at the moment.
The theory on which this is based upon can be read in the "Procedural Textures by Tiling and Blending for Displacement Mapping.pdf" if you're more interested in the mathematical part.
If you are interested in TFDM, there is an implementation in "raytrace.rint" (lines 94-303).

In conclusion the project is for now on ice for an undefined period of time (as I have some studies to do), you might want to come back at a later point in time.

**Comment by ib00** [comment]
Thanks! TFDM paper seems very tricky to implement, so I can imagine it was stressful.
